bin/check-http.rb

```diff
@@ -251,9 +251,11 @@ def run
     end
 
     begin
-      Timeout.timeout(config[:timeout]) do
```

Comment:
Inside `acquire_resource` below we should use [this](https://ruby-doc.org/stdlib-2.4.1/libdoc/net/http/rdoc/Net/HTTP.html#attribute-i-read_timeout) to still provide a way for people to account for slow overloaded web servers.